src/cargo/core/compiler/context/unit_dependencies.rs

```diff
@@ -273,28 +283,65 @@ fn maybe_lib<'a, 'cfg>(unit: &Unit<'a>, cx: &Context<'a, 'cfg>) -> Option<Unit<'
 /// script itself doesn't have any dependencies, so even in that case a unit
 /// of work is still returned. `None` is only returned if the package has no
 /// build script.
-fn dep_build_script<'a, 'cfg>(unit: &Unit<'a>, cx: &Context<'a, 'cfg>) -> Option<Unit<'a>> {
+fn dep_build_script<'a>(cx: &Context, unit: &Unit<'a>) -> Option<(Unit<'a>, ProfileFor)> {
```

Comment:
Some function in this module now use cx, unit arg order, and some use (unit, cx). Let's unify them to one order? I don't have any preference for a specific order, as long as it is an order :-)

Reply:
Yea, I derped.  I had an earlier version that I had removed Context, but when I added it back I should have kept it the same.